modules/ROOT/pages/batch-error-handling-faq.adoc

```diff
@@ -1,23 +1,33 @@
-= Handling Errors During Batch Job
+= Handling Errors During Batch Jobs
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 == Overview
 
-Handling errors inside a batch job can be complicated considering the huge amount of data this scope is designed to take. +
-When processing 1 million records, it's almost impossible to log everything. Logs would become huge, unreadable, and would take a massive toll on the performance of your processing. +
-On the other hand, logging too little would make it impossible for you to know what went wrong. And considering that if out of those 1 million records, 30 thousand of them failed, not knowing what’s wrong with them can turn your debugging task very problematic. This trade-off is not simple to overcome.
+Mule batch processing is designed to handle very large data sets and to perform "near real-time" data integration that recovers from crashes and continues processing a job from a point of failure. However, _verbose_ logs for issues that occur can become enormous and severely impact performance. 
 
+To limit this impact, Mule uses INFO-level logging by default, as described in <<default_logging>>. For cases in which you require more verbose log messages, you can change the mode to DEBUG. This mode is helpful for debugging and feasible for some cases that involve smaller data sets. 
+
+The following property is available for setting the logging mode:
+
+[source,text,linenums]
+----
+log4j.logger.com.mulesoft.module.batch=DEBUG
+----
+
+Avoid using the DEBUG mode in a production environment when processing very large datasets. 
+
+[[default_logging]]
 == Logs of Failing Records Inside a Batch Step
 
-When processing a batch job instance, a processor inside a batch step may fail or raise an error. When this occurs – perhaps because of corrupted or incomplete record data – Mule logs the stack traces following this logic:
+When processing a batch job instance, a processor inside a batch step can fail or raise an error, for example, because of corrupted or incomplete record data. By default, Mule logs the stack traces when such an issue occurs by following this logic:
```

Comment:
```suggestion
When processing a batch job instance, a processor inside a batch step can fail or raise an error, for example, because of corrupted or incomplete record data. By default, Mule uses the INFO log level to log stack traces according to the following logic when issues occur:
```